src/utils.ts

```diff
@@ -13,3 +16,67 @@ export const aestheticBaseAttributes: Pick<Nft, 'aestheticRank' | 'aestheticRank
     timesWon: 0,
     aestheticRankReliability: 0,
 };
+
+/**
+ * A mapping of all PopRank supported chains to all
+ * of our supported marketplaces.
+ *
+ * The first marketplace under a chain is considered the "default".
+ */
+export const chainMarketplaceMap: ChainMarketplaceMap = {
+    ethereum: {
+        opensea: {
+            name: "OpenSea",
+            nftUrlFormatter: (address?: string, id?: string) => `${OPENSEA_BASE_NFTS_URL}/${address}/${id}`,
+            collectionUrlFormatter: (address: string) => `${OPENSEA_BASE_COLLECTIONS_URL}/${address}`
+        }
+    },
+    polygon: {
+        opensea: {
+            name: "OpenSea",
+            nftUrlFormatter: (address?: string, id?: string) => `${OPENSEA_BASE_NFTS_URL}/matic/${address}/${id}`,
+            collectionUrlFormatter: (address: string) => `${OPENSEA_BASE_COLLECTIONS_URL}/${address}`
+        }
+    },
+    solana: {
+        magiceden: {
+            name: "MagicEden",
+            nftUrlFormatter: (_address?: string, id?: string) => `${MAGICEDEN_BASE_NFTS_URL}/${id}`,
+            collectionUrlFormatter: (address: string) => `${MAGICEDEN_BASE_COLLECTIONS_URL}/${address}`
+        },
+        opensea: {
+            name: "OpenSea",
+            nftUrlFormatter: (address?: string, id?: string) => `${OPENSEA_BASE_NFTS_URL}/solana/${address}/${id}`,
+            collectionUrlFormatter: (address: string) => `${OPENSEA_BASE_COLLECTIONS_URL}/${address}`
+        }
+    },
+};
+
+/** Holds the block explorer URL account/address lookup prefix for each chain */
+export const blockExplorerAddressLinkFormatters: Record<Chain, (identifier: string) => string> = {
```

Comment:
is this object meant to be used in `chainBlockExplorerMap`?

Reply:
deleted now, it was the old object used by the frontend to format links, before i applied the same marketplace structure to explorers